spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/GoogleSecretManagerEnvironmentProperties.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
```

Comment:
update files touched in the PR to `2020` instead `2019`

Reply:
I think I did that to all now.